Lesson6_2/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿#region Task2
+/*
+ Person's Class -  has a   name and age. The  Сitizen subsidiary(inherited from Person), that implements the IHasId interface 
+- has an identifier code -  the string Id {get;} property. Another class -  LegalPerson (legal entity) - also implements the IHasId interface, 
+but is not inherited from Person. Class Bank -  has methods void 
+PutMoneyToAccount (IHasId accountHolder, int quantity) - put money on the account
+void GetMoneyFromAccount (IHasId accountholder, int quantity)- withdraw money 
+from an account int GetAccountBalance (IHasId accountHolder)  -  get an account 
+balance Check the bank for both Citizen instances and LegalPerson.
```

Comment:
Check the bank for both Citizen instances and LegalPerson. - this requirement is not met ❌ 
It requires you to write inside Program class checks for type of instances aggregated in bank ecnapsulated collection. And encapsulate those instances inside bank as some kind of collection.